pkg/limayaml/limayaml.go

```diff
@@ -229,6 +230,12 @@ type Provision struct {
 	Script                          string             `yaml:"script" json:"script"`
 	File                            *LocatorWithDigest `yaml:"file,omitempty" json:"file,omitempty" jsonschema:"nullable"`
 	Playbook                        string             `yaml:"playbook,omitempty" json:"playbook,omitempty"`
+	// The following fields must be nil unless Mode is ProvisionModeData
+	Content     *string `yaml:"content,omitempty" json:"content,omitempty" jsonschema:"nullable"`
+	Overwrite   *bool   `yaml:"overwrite,omitempty" json:"overwrite,omitempty" jsonschema:"nullable"`
+	Owner       *string `yaml:"owner,omitempty" json:"owner,omitempty"`
```

Comment:
The format should be clarified in a godoc comment. Does this accept numeric UIDs? Does this accept specifying groups? ("USER:GROUP")